services/service-core/src/auth/keycloak.service.ts

```diff
@@ -16,9 +16,16 @@ import axios from 'axios';
 import { ConfigService } from '@nestjs/config';
 import { Config } from './config';
 import { UpdateUser } from 'src/graphql/users/users.entity';
+import { userInputValidator } from './keycloak.validator';
 
 @Injectable()
 export class KeycloakService {
+  getAdminTokenParams() {
```

Comment:
What's the point in having these two methods that do nothing and only throw exceptions? Seems like a YAGNI violation